### PR TITLE
SET-81

### PR DESCRIPTION
**Pull request by iweiss**
This PR brings the EAP component versions to Fuse in order to achieve alignment between both products

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -75,15 +75,29 @@
...
+        <version.org.fusesource.activemq>7.0.0.fuse-000211-redhat-1</version.org.fusesource.activemq>
+        <version.org.fusesource.camel-sap>7.0.0.fuse-000211-redhat-1</version.org.fusesource.camel-sap>
         <version.org.apache-extras.camel-extra>2.21.0.fuse-000054</version.org.apache-extras.camel-extra>
 
+        <!--JBoss EAP components versions. This must be kept in sync with whatever comes from their BOMs -->
+        <version.io.undertow>1.4.18.SP7-redhat-1</version.io.undertow>
+        <version.joda-time>2.9.7.redhat-1</version.joda-time>
+        <version.org.bouncycastle>1.56.0.redhat-2</version.org.bouncycastle>
+        <version.org.hibernate>5.1.13.Final-redhat-1</version.org.hibernate>
+        <version.org.jboss.eap>7.1.2.GA-redhat-1</version.org.jboss.eap>
```
Comment (wolfc) [inline pom.xml]: Obsolete :)

**Comment by cunningt** [comment]
Should we be taking this one or close it?   It's been sitting out there for a while.

**Comment by wolfc** [comment]
If you do wish to go with it, then it should be redone for EAP 7.1.5.

**Comment by cunningt** [comment]
This is very out-of-date and I am going to close it.    If it's still applicable, please resubmit against the 7-x branch.

**Comment by grgrzybek** [comment]
please don't resubmit. There are many special overrides that would have to be done in Fuse Karaf's pom. I even spent some time trying to get parent POM from EAP, but it just didn't work.

**Pull request closed without merging**